src/azure-cli-core/azure/cli/core/__init__.py

```diff
@@ -499,7 +499,7 @@ def load_arguments(self, command=None):
             command_loaders = self.cmd_to_loader_map.get(command, None)
 
         if command_loaders:
-            for loader in command_loaders:
+            for loader in sorted(command_loaders, key=lambda x:x.__class__.__name__):
```

Comment:
Some questions we can discuss:

- Why does docs depends on the order of command loaders?
- Are there any other solutions which may be better?
- Is there any performance downgrade if we sort command loaders?